auth.go

```diff
@@ -30,8 +30,21 @@ func (me *PlainAuth) Response() string {
 	return fmt.Sprintf("\000%s\000%s", me.Username, me.Password)
 }
 
+// CertAuth for RabbitMQ-auth-mechanism-ssl.
+type CertAuth struct {
+}
+
+func (me *CertAuth) Mechanism() string {
```

Comment:
And for naming convention again, @streadway seems to use `auth` instead of `me`.